infrastructure/terraform/README.md

```diff
@@ -0,0 +1,29 @@
+# OpenTofu based deployment
+
+## Requirements
+
+- OpenTofu
+- GCloud CLI
+- GCP Project
+
+## Bootstrap
+
+To bootstrap the account
```

Comment:
I feel like we are missing some context here. Can you please clarify what bootstrap means in this context, when and how it should be executed?

Reply:
I added the steps to apply changes in each module.